src/router/controller/params/decorators.ts

```diff
@@ -35,6 +35,12 @@ export const Res = createParamDecorator((context: HttpContext) => {
 	return context.response;
 });
 
+export const Header = (prop?: string) =>
+	createParamDecorator((context: HttpContext) => {
+		if (!context.request.headers) return null;
```

Comment:
The actual code style require braces around all if statement, even single line (you can see that below in other functions)